### ISSUE TITLE
TypeError: openapi.servers is not iterable

### ISSUE THREAD

**Issue by jboxman** [Mermade/oas-kit#171]
I'm not sure what this means.

```
$ swagger-cli validate ../openshift-openapi-spec.json
../openshift-openapi-spec.json is valid
```

### Detailed Description

```
TypeError: openapi.servers is not iterable
    at processPaths (/Users/jasonb/Self/work/apis/node_modules/widdershins/node_modules/swagger2openapi/index.js:863:56)
    at main (/Users/jasonb/Self/work/apis/node_modules/widdershins/node_modules/swagger2openapi/index.js:1059:5)
    at /Users/jasonb/Self/work/apis/node_modules/widdershins/node_modules/swagger2openapi/index.js:1366:13
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
openapi.servers is not iterable
TypeError: openapi.servers is not iterable
    at processPaths (/Users/jasonb/Self/work/apis/node_modules/widdershins/node_modules/swagger2openapi/index.js:863:56)
    at main (/Users/jasonb/Self/work/apis/node_modules/widdershins/node_modules/swagger2openapi/index.js:1059:5)
    at /Users/jasonb/Self/work/apis/node_modules/widdershins/node_modules/swagger2openapi/index.js:1366:13
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:743:3)
```

**Comment by MikeRalphson** [comment]
Can you send me a link or a gist of the document you're trying to validate?

**Comment by jboxman** [comment]
Sure, it's here:

https://github.com/openshift/origin/blob/release-4.1/api/swagger-spec/openshift-openapi-spec.json

**Comment by MikeRalphson** [comment]
Thanks! (https://raw.githubusercontent.com/openshift/origin/release-4.1/api/swagger-spec/openshift-openapi-spec.json)

**Comment by MikeRalphson** [comment]
This code has already been fixed in swagger2openapi, and so just waiting on the next release of widdershins to pull it in.

**Comment by jboxman** [comment]
Thanks @MikeRalphson!

**Comment by MikeRalphson** [comment]
Should be fixed in `widdershins` version `4.0.0`, which includes breaking changes (please see the `README.md` file).
